homeassistant/components/sensor/envirophat.py

```diff
@@ -157,38 +161,37 @@ def __init__(self, use_leds):
         self.voltage_1 = None
         self.voltage_2 = None
         self.voltage_3 = None
+        self.update()
```

Comment:
Because of the `@Throttle`, looks like it needs `add_device()` without `True`, and adding the `update()` call in the sensor's init (and I've seen that's how it is in other throttled sensors in general, now it makes sense).